.gitignore

```diff
@@ -7,6 +7,8 @@ cfg/*
 
 # Ignore data/
 data/
+!lib/Data
```

Comment:
I assume these had to be added due to an OS using case-insensitive file system? Would it maybe be cleaner to rewrite the rule before this one to an absolute path (`/data/` instead of `data/`), so it doesn't match the {lib,tst}/Data folders?

Reply:
done. I changed it to /data/.